### DISCUSSION TITLE
Welcome to subtitle-translator-electron Discussions!

### DISCUSSION THREAD

**Discussion by gnehs** [gnehs/subtitle-translator-electron#87] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leochoo** [comment]
Awesome!

**Comment by tokyoanil** [comment]
When will the GPT4o model be implemented?

**Reply by gnehs** [reply]
It can be used now, but the price calculation will be fixed in the next update.

**Reply by tokyoanil** [reply]
Thanks sir, In the model section it says GPT4 Turbo and only economic mode and expert mode are available.  When we use the model that we do not put in economic mode and it says GPT4 Turbo, is it GPT4o? When will the update for price calculation come dusty?

**Reply by gnehs** [reply]
Just enter `gpt-4o` in the model name field, and you're all set!

<img width="801" alt="image" src="https://github.com/gnehs/subtitle-translator-electron/assets/16719720/e30fe1cb-b9f0-403e-b531-ce998a18531f">

**Reply by tokyoanil** [reply]
Thank you. But, although I added 5 api keys, whenever I do language translation, 5 6 lines are not translated, how can I overcome this?

**Reply by tokyoanil** [reply]
?
